packages/core-mobile/app/contexts/DeeplinkContext/DeeplinkContext.tsx

```diff
@@ -39,78 +40,95 @@ export const DeeplinkContextProvider = ({
   children: React.ReactNode
 }): JSX.Element => {
   const dispatch = useDispatch()
+  const store = useStore()
   const walletState = useSelector(selectWalletState)
   const isWalletActive = walletState === WalletState.ACTIVE
   const isDeveloperMode = useSelector(selectIsDeveloperMode)
   const isNotificationBlocked = useSelector(selectIsNotificationBlocked)
   const processedFeatureFlags = useSelector(selectFeatureFlags)
   const [pendingDeepLink, setPendingDeepLink] = useState<DeepLink>()
 
-  const expireDeepLink = useCallback(() => {
-    setPendingDeepLink(undefined)
-  }, [])
+  const maybeSetActiveNetwork = useCallback(
+    (data: NotificationData) => {
+      if (
+        'chainId' in data &&
+        ['string' || 'number'].includes(typeof data.chainId)
```

Comment:
should this be ['string, 'number]?

Reply:
yes! eagle eye
[Fix array and data type](https://github.com/ava-labs/core-mobile/pull/1659/commits/b1fb940934027accbc3947c307add309a5f5bf41)